internal/handler/impl/host_handler.go

```diff
@@ -13,11 +13,31 @@ import (
 	"gorm.io/gorm"
 )
 
+const (
+	pendoHostConfSuccess = "HostConf"
```

Comment:
I'm thinking about name spacing the track event. e.g. `idmsvc-host-conf` or `idmsvc-HostConf`, wdyt?

Reply:
Given the two options above, I'd rather `idmsvc-host-conf` to `idmsvc-HostConf`; rational: the second mix kebab case and pascal case.

* snake case: `number_of_donuts` or `NUMBER_OF_DONUTS`.
* kebab case: `number-of-donuts` or `NUMBER-OF-DONUTS`.
* camel case: `numberOfDonuts`.
* pascal case: `NumberOfDonuts`.

https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/

Looking at pendo, Track Events, I would like to know where would this value is mapped; if this value would be mapped to the `Name` column I can see a mix of different naming conventions; so I would say:
* I would ask to analytics mates if there are some naming convention to be aligned with it.
* If not naming convention defined or used and we are free to choose the one that fit better for us, I would say `kebab-case` as I have seen many events with the Name column using that convention; additionally, we could define scope as we progress (for instance, `idmsvc-host-conf-success` or `idmsvc-host-conf-failure`), and IMHO I would add a final label to indicate what event is happening: `success` or `failure`.

Based on the last example `idmsvc-hostconf-{success,failure}` the pattern would looks like: `<service>-<label-path-of-my-event>-<label-to-give-semantic-to-the-event>`; examples:
* `idmsvc-host-conf-success`, where:
  * `service` = `idmsvc`
  * `label-path-of-my-event` = `host-conf` or `auto-enroll-host-conf` or `ipa-auto-enroll-host-conf`
  * `label-to-give-semantic-to-the-event` = `success` or `failure`

Reply:
Fixed, used `idmsvc-host-conf-success` and `idmsvc-host-conf-failure`.